LeaderboardBackend.Test/Leaderboards.cs

```diff
@@ -165,18 +165,53 @@ public async Task GetLeaderboards_Deleted_BySlug_NotFound()
         {
             Name = "Should 404",
             Slug = "should-404",
-            UpdatedAt = _clock.GetCurrentInstant() + Duration.FromMinutes(1),
-            DeletedAt = _clock.GetCurrentInstant() + Duration.FromMinutes(1),
+            UpdatedAt = _clock.GetCurrentInstant() - Duration.FromMinutes(1),
+            DeletedAt = _clock.GetCurrentInstant() - Duration.FromMinutes(1),
         };
 
         context.Leaderboards.Add(board);
         await context.SaveChangesAsync();
-        _clock.AdvanceMinutes(2);
 
         Func<Task<LeaderboardViewModel>> act = async () => await _apiClient.Get<LeaderboardViewModel>($"/api/leaderboard?slug={board.Slug}", new());
         await act.Should().ThrowAsync<RequestFailureException>().Where(e => e.Response.StatusCode == HttpStatusCode.NotFound);
     }
 
+    [Test]
+    public async Task DeletedBoardsDontConsumeSlugs()
+    {
+        ApplicationContext context = _factory.Services.CreateScope().ServiceProvider.GetRequiredService<ApplicationContext>();
+
+        Leaderboard deletedBoard = new()
+        {
+            Name = "Super Mario World (OLD)",
+            Slug = "super-mario-world",
+            DeletedAt = _clock.GetCurrentInstant()
+        };
+
+        context.Leaderboards.Add(deletedBoard);
+        await context.SaveChangesAsync();
+        deletedBoard.Id.Should().NotBe(default);
```

Comment:
What does default resolve to? new Guid()?

Reply:
It's a `long` so zero. This is just a more idiosyncratic way of expressing it.